WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -53,8 +48,22 @@ struct AccountSettingsService {
         }
     }
 
+    /// Emits a value when the settings for the associated account change.
+    var settings: Observable<AccountSettings?> {
+        let notificationCenter = NSNotificationCenter.defaultCenter()
+        let notificationObserver = notificationCenter.rx_notification(NSManagedObjectContextDidSaveNotification, object: context)
+        // This was the simplest implementation. If performance is an issue, we could try
+        // adding `distinctUntilChanged` or `filter` on the notification userInfo and only
+        // emit if the changed objects include the observed account.
+        return notificationObserver.map(getSettings).startWith(getSettings())
```

Comment:
Actually no, in #4732 the refresh observable is only subscribed to while the view is visible, but the data is subscribed until the controller is deallocated